src/packages/excalidraw/publicPath.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable */
+/* global __webpack_public_path__:writable */
+__webpack_public_path__ =
+  window.EXCALIDRAW_ASSET_PATH ||
+  "https://unpkg.com/aakansha-excalidraw@0.4.0-asset/dist/";
```

Comment:
`https://unpkg.com/aakansha-excalidraw@0.4.0-asset/dist/` prob should be replaced with our own :)

Reply:
Actually, what's the purpose of this file?

Reply:
yep this was for testing :).